lib/bolt/cli.rb

```diff
@@ -313,6 +313,7 @@ def parse
       validate(options)
 
       # After validation, initialize inventory and targets. Errors here are better to catch early.
+      @logger.debug("Get targs #{inventory.method(:get_targets)}")
```

Comment:
Why would we call this twice?

Reply:
Woops, leftover debugging line. Removed.